v1/transform-global.yaml

```diff
@@ -0,0 +1,227 @@
+info:
+  version: '1.0.0'
+  title: Global Transform API
+  description: Global Transform API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /html/{from}/to/{to}{/provider}:
```

Comment:
>  To be consistent with the other transform end points, how about `transform/html/from/{from_lang}/to/{to_lang}{/provider}` ?

The structure is slightly different, as other end points tend to be organized around `/{from_format}/to/{to_format}/`. The new endpoint is not converting the format, but the language within a given format (HTML).

I think the renamed parameters could work. I slightly prefer the look and feel of `/transform/html-lang/{from}/to/{to}{/provider}`, but don't feel too strongly about it. It seems to more clearly communicate what is being transformed (`html-lang`).